src/features/projects/styles/Projects.scss

```diff
@@ -1,5 +1,5 @@
 @import './../../../theme/theme';
-@import '../../projectsV2/styles/ProjectSnippet.module.scss';
+@import '../../projectsV2/ProjectSnippet/styles/ProjectSnippet.module.scss';
 @import './ProjectDetails.module.scss';
```

Comment:
Why are we using v2 styles within the old component?

Reply:
I removed the import since none of the classes from `ProjectSnippet.module.scss `were being used in `project.scss`